pkg/apiserver/resthandler.go

```diff
@@ -519,3 +534,22 @@ func setListSelfLink(obj runtime.Object, req *restful.Request, namer ScopeNamer)
 	return runtime.SetList(obj, items)
 
 }
+
+func getPatchedJS(contentType string, originalJS, patchJS []byte, obj runtime.Object) ([]byte, error) {
```

Comment:
Our current code organization has a lot of problems. Comment on #4851 if you have ideas. Code needed by the client and server should be factored out into a common/shared package.